.github/workflows/netlify-deploy-v2.yaml

```
@@ -126,6 +126,7 @@ jobs:
         working-directory: ./docs
         env:
           HUGO_ENV: production
+          hugo_env: public
```

actionlint (1.7.12)

[error] key "hugo_env" is duplicated in "env" section. previously defined at line:128,col:11. note that this key is case insensitive

(syntax-check)